FND/Places/Addresses.rdf

```diff
@@ -763,6 +763,12 @@
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
 	</owl:ObjectProperty>
 	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-adr;hasLocaleDesignation">
```

Comment:
The name is pretty misleading - it's not a designation for the locale, but a locale-specific designation for something else.
Also "designation" clashes with the use of the term elsewhere in FIBO (which I don't like but we should be consistent) for appointing someone to a position.